gulp/tasks/report.js

```diff
@@ -1,12 +1,16 @@
 var gulp = require('gulp');
 var paths = require('../util/paths');
 
-gulp.task('report',['start'],function(){
+gulp.task('report', function(){
 
   if (!paths.report || paths.report.indexOf( 'browser' ) > -1 ){
     setTimeout(function(){gulp.run('openReport')},100);
   }
 
+  if (!paths.report || paths.report.indexOf( 'prepare' ) > -1 ){
+    setTimeout(function(){gulp.run('createReport')},100);
+  }
+
   if (!paths.report || paths.report.indexOf( 'CLI' ) > -1 ){
     setTimeout(function(){gulp.run('compare')},1000);
```

Comment:
I'm not sure why we need the timeouts for the CLI/prepare reporters. I'm pretty sure we can run them without delay.